js/src/index.js

```diff
@@ -24,3 +25,17 @@ addFilter( 'woocommerce_admin_reports_list', 'analytics/scheduled-actions', repo
 	  },
   ];
 } );
+
+/**
+ * Use the 'woocommerce_admin_time_excluded_screens' filter to remove date parameters from the query string.
+ */
+addFilter( 'woocommerce_admin_reports_list', 'analytics/scheduled-actions', pages => {
```

Comment:
This is the same filter as the block above. Is this supposed to be the `woocommerce_admin_time_excluded_screens` filter?

If so, you'll just need to push `scheduled-actions` onto the array before returning it.

Reply:
This change should have been in #11. I have updated it here to use the correct filter and changed the unique filter names to better names.